localization/src/index.ts

```diff
@@ -1,3 +1,5 @@
+import './DayJSConfig.js';
```

Comment:
```suggestion
import './DayJSConfig';
```

Reply:
Also, since you've moved it from @packages/core, let's kill it there (as it's not needed anymore)